src/common/meta/src/key/flow/flow_name.rs

```diff
@@ -162,6 +178,25 @@ impl FlowNameManager {
             .transpose()
     }
 
+    /// Return all flows' names and ids
+    pub async fn flow_names(
+        &self,
+        catalog: &str,
+    ) -> BoxStream<'static, Result<(String, FlowNameValue)>> {
+        let start_key = FlowNameKey::range_start_key(catalog);
+        common_telemetry::debug!("flow_names: start_key: {:?}", start_key);
+        let req = RangeRequest::new().with_prefix(start_key.as_bytes());
+
+        let stream = PaginationStream::new(
+            self.kv_backend.clone(),
+            req,
+            DEFAULT_PAGE_SIZE,
+            Arc::new(flow_name_decoder),
+        );
+
+        Box::pin(stream)
+    }
```

Comment:
Let's add a unit test.

Reply:
> Let's add a unit test.

added a test for `range_start_key` since I didn't found good example for directly testing functions like `flow_names`(or `schema_names()`